packages/react-compass/scripts/build.ts

```diff
@@ -16,42 +17,95 @@ const makeCliOptions = (o: Record<string, unknown>) =>
     .map(([key, value]) => `--${key} ${value}`)
     .join(' ')
 
-// Define Tasks
+const getTypesVersions = async () => {
+  const result: any = {}
+  const exclude = ['utils']
+  const capitalize = (s: string) => s.charAt(0).toUpperCase() + s.slice(1)
+  const folders = await fs.readdir(OUTPUT_ESM_DIR, {withFileTypes: true})
+  const directoriesInDIrectory = folders
+    .filter((item) => item.isDirectory())
+    .map((item) => item.name)
+  directoriesInDIrectory.forEach((item) => {
+    if (!exclude.includes(item)) {
+      result[capitalize(item)] = [
```

Comment:
we might want to triage on whether we want to support capitalized imports.

Reply:
i think we need to support both ways because someone familiar with capitalize, it look like a component when used

Reply:
Can you remove it please, it might be confusing for the users, and the auto-import on IDEs may use either of them randomly, which can be bad. As discussed with @TaiCBui.